util/cobrautil/cobrautil.go

```diff
@@ -12,3 +16,38 @@ func HideInheritedFlags(cmd *cobra.Command, hidden ...string) {
 		_ = cmd.Flags().MarkHidden(h)
 	}
 }
+
+const annotationExperimentalCLI = "experimentalCLI"
+
+func MarkFlagExperimental(f *pflag.Flag) {
+	if _, ok := f.Annotations[annotationExperimentalCLI]; ok {
+		return
+	}
+	if f.Annotations == nil {
+		f.Annotations = make(map[string][]string)
+	}
+	f.Annotations[annotationExperimentalCLI] = nil
+	f.Usage += " (EXPERIMENTAL)"
```

Comment:
I wonder if it would be better to handle this with a custom help template or function, rather than augmenting the cmd.Usage
https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#defining-your-own-help

Reply:
Maintenance cost can be higher if we switch away from the default template